### PR TITLE
disable interstitial for master

### PR DESCRIPTION
**Pull request by danteruiz**

### CODE REVIEW

**Comment by conklin94122** [comment]
We do not yet know if we are disabling or enabling interstitial for 76.0, but I want this PR ready and tested just in case we are leaving it disabled. Please do not merge this yet.

**Comment by conklin94122** [comment]
This is clear to be tested, reviewed, and merged when ready.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15173/HighFidelity-Beta-PR14406-f17723c.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15173/HighFidelity-Beta-PR14406-f17723c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15173/HighFidelity-Beta-Interface-PR14406-f17723c.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15173/HighFidelity-Beta-PR14406-f17723c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15173/HighFidelity-Beta-Interface-PR14406-f17723c.exe

File: libraries/networking/src/DomainHandler.h [hunk 1]
```diff
@@ -234,7 +234,7 @@ private slots:
 #ifdef Q_OS_ANDROID
     Setting::Handle<bool> _enableInterstitialMode{ "enableInterstitialMode", false };
 #else
-    Setting::Handle<bool> _enableInterstitialMode { "enableInterstitialMode", true };
```
Comment (SamGondelman) [inline libraries/networking/src/DomainHandler.h]: Wouldn't this only disable it on clean installs?

Comment (ZappoMan) [reply libraries/networking/src/DomainHandler.h]: hmmmm - if that's true we still have a problem for 75!!

Comment (ZappoMan) [reply libraries/networking/src/DomainHandler.h]: stop scaring me @SamGondelman !

Comment (SamGondelman) [reply libraries/networking/src/DomainHandler.h]: never

**Comment by danteruiz** [comment]
Yes, I could create a custom script to toggle interstitial mode, either by menu or tablet. This scripts should hosted, not in our script directory.

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Interstitial no longer appears

**Pull request merged**